diff --git a/base/BUILD.gn b/base/BUILD.gn
index 8bd6530db838c9e44cbf04a2db7b59c9a435654f..327d2b1bffdb597e5772ffa49614cc645767da3d 100644
--- a/base/BUILD.gn
+++ b/base/BUILD.gn
@@ -4461,6 +4461,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/task/ThreadPoolTaskExecutor.java",
       "android/java/src/org/chromium/base/task/UiThreadTaskExecutor.java",
     ]
+    sources += brave_java_base_sources
 
     if (use_clang_profiling) {
       sources += [
@@ -4499,6 +4500,7 @@ if (is_android) {
       "android/java/src/org/chromium/base/shared_preferences/SharedPreferencesManager.java",
       "android/java/src/org/chromium/base/shared_preferences/StrictPreferenceKeyChecker.java",
     ]
+    sources += brave_base_shared_preferences_java_sources
   }
 
   android_aidl("process_launcher_aidl") {
