diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index ecc7f24598744d0e249629695ad1fb476bd8fdc4..f66ef01825a1f22215fbb98341a6398acd8adeb6 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2663,6 +2663,7 @@ static_library("browser") {
       "performance_monitor/metric_evaluator_helper_posix.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps allow_circular_includes_from += brave_chrome_browser_allow_circular_includes_from
 
   if (!is_chromeos_ash) {
     sources += [
