diff --git a/components/sync/service/sync_service_impl.h b/components/sync/service/sync_service_impl.h
index 9af8787e29ab71a8bc14be29346bbf03c5fdf763..ebea6d9100d87cf010084e531023be50ce4f766d 100644
--- a/components/sync/service/sync_service_impl.h
+++ b/components/sync/service/sync_service_impl.h
@@ -259,6 +259,7 @@ class SyncServiceImpl : public SyncService,
   SyncClient* GetSyncClientForTest();
 
  private:
+  BRAVE_SYNC_SERVICE_IMPL_H_
   enum UnrecoverableErrorReason {
     ERROR_REASON_ENGINE_INIT_FAILURE,
     ERROR_REASON_ACTIONABLE_ERROR,
