diff --git a/components/update_client/BUILD.gn b/components/update_client/BUILD.gn
index 053d0220a86cb0e46381e4e4e50bdacacc587869..49635975c58a8ca67fd1299270ca8d2c6c3c8261 100644
--- a/components/update_client/BUILD.gn
+++ b/components/update_client/BUILD.gn
@@ -181,6 +181,7 @@ static_library("update_client") {
       "background_downloader_win.h",
     ]
   }
+  deps += brave_components_update_client_deps public_deps = brave_components_update_client_public_deps
 }
 
 source_set("test_support") {
