diff --git a/content/public/browser/content_browser_client.h b/content/public/browser/content_browser_client.h
index 2549daafcb3c39642015efa0538fb4ace29756c9..9ea20b428b7da27d04ad9f2ffe84d84c4ef5fb62 100644
--- a/content/public/browser/content_browser_client.h
+++ b/content/public/browser/content_browser_client.h
@@ -2312,6 +2312,7 @@ class CONTENT_EXPORT ContentBrowserClient {
   virtual ui::AXMode GetAXModeForBrowserContext(
       BrowserContext* browser_context);
 
+  BRAVE_CONTENT_BROWSER_CLIENT_H
 #if BUILDFLAG(IS_ANDROID)
   // Defines the heuristics we can use to enable wide color gamut (WCG).
   enum class WideColorGamutHeuristic {
