diff --git a/third_party/blink/renderer/core/frame/local_dom_window.cc b/third_party/blink/renderer/core/frame/local_dom_window.cc
index de9377575794e93864db43b59807e3af6c16f33d..45d5ff59cbdb8f3afe764ecef2a03c5e45394171 100644
--- a/third_party/blink/renderer/core/frame/local_dom_window.cc
+++ b/third_party/blink/renderer/core/frame/local_dom_window.cc
@@ -499,6 +499,7 @@ bool LocalDOMWindow::CanExecuteScripts(
   bool allow_script_content_setting =
       GetFrame()->GetContentSettings()->allow_script;
   bool script_enabled = allow_script_renderer && allow_script_content_setting;
+  BRAVE_LOCAL_DOM_WINDOW_CAN_EXECUTE_SCRIPTS
   if (!script_enabled && reason == kAboutToExecuteScript) {
     WebContentSettingsClient* settings_client =
         GetFrame()->GetContentSettingsClient();
